java/java/JavaParser.g4

```diff
@@ -377,6 +410,12 @@ blockStatement
 
 localVariableDeclaration
     : variableModifier* typeType variableDeclarators
+    | variableModifier* VAR identifier '=' expression
```

Comment:
Alternatives can be merged:

```suggestion
    : variableModifier* (typeType variableDeclarators | VAR identifier '=' expression)
```

It's important because the first part can match the empty string, which may affect performance.